test/unit/amqpBinding-vhost-test.js.js

```diff
@@ -0,0 +1,223 @@
+/*
```

Comment:
test/unit/amqpBinding-vhost-test.js.js -> test/unit/amqpBinding-vhost-test.js

Too much "js" at the end ;)